src/SonarScanner.MSBuild.PreProcessor/TeamBuildPreProcessor.cs

```diff
@@ -254,6 +263,20 @@ private async Task<ArgumentsAndRuleSets> FetchArgumentsAndRulesets(ISonarQubeSer
             return argumentsAndRuleSets;
         }
 
+        private static bool IsPullRequest(ProcessedArgs localSettings, out string baseBranch)
```

Comment:
Changed signature to remove `out` argument
Renamed argument, it's not important here that it's the local version.
```suggestion
        private static string PullRequestBaseBranch(ProcessedArgs settings)
```